API/chat/tests/sessions/common.py

```diff
@@ -0,0 +1 @@
+from chat.tests.common import *
```

Comment:
do not use `*`

Reply:
This logic with "*" is used in the other tests(not created by me) as well.